homeassistant/components/ituran/quality_scale.yaml

```diff
@@ -0,0 +1,89 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration does not explicitly subscribe to events.
+  dependency-transparency: done
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  docs-actions:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  brands: done
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  reauthentication-flow: done
+  parallel-updates:
+    status: exempt
+    comment: |
+      Read only platforms and coordinator.
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters: done
+  docs-configuration-parameters:
+    status: exempt
+    comment: |
+      No options flow.
+  # Gold
+  entity-translations: done
+  entity-device-class:
+    status: exempt
+    comment: |
+      Only device_tracker platform.
+  devices: done
+  entity-category: todo
+  entity-disabled-by-default: todo
```

Comment:
```suggestion
  entity-disabled-by-default:
    status: exempt
    comment: |
      No noisy entities
```

Reply:
I'm going along with this but it's a bit weird as this (and some others you suggested I mark as exempt) say there are no exceptions to the rule, e.g.: https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/entity-disabled-by-default#exceptions

Reply:
I know, but you only have one entity so there’s nothing you could disable. Also the docs still are pretty new and need some updates here and there